core/src/main/java/bisq/core/api/CoreTradesService.java

```diff
@@ -303,16 +312,18 @@ Trade getTrade(String tradeId) {
                     .map(t -> (TradeModel) t)
                     .collect(Collectors.toList());
             closedTrades.addAll(bsqSwapTradeManager.getBsqSwapTrades());
-            // TODO Sort closedTrades by date?
-            return closedTrades;
+            return closedTrades.stream().sorted(dateComparator.get()).collect(Collectors.toList());
         } else {
             var failedV1Trades = failedTradesManager.getTrades();
-            return failedV1Trades.stream().map(t -> (TradeModel) t).collect(Collectors.toList());
+            return failedV1Trades.stream()
+                    .map(t -> (TradeModel) t)
```

Comment:
Why do you need to map it here?

Reply:
It is something @xyz123 asked for in his review: "why isn't the mapping consistent between return types for List<TradeModel> getOpenTrades() and List<TradeModel> getTradeHistory(...)"? I did intend make the return types consistent, but overlooked it.